apps/admin-ui/src/clients/authorization/NewPolicyDialog.tsx

```diff
@@ -30,6 +32,12 @@ export const NewPolicyDialog = ({
   toggleDialog,
 }: NewPolicyDialogProps) => {
   const { t } = useTranslation("clients");
+  const localeSort = useLocaleSort();
+
+  const sortedPolicies = useMemo(
+    () => localeSort(policyProviders!, mapByKey("name")),
```

Comment:
It's possible for `policyProviders` to be `undefined`. Let's try to take that into account.
```suggestion
    () => policyProviders ? localeSort(policyProviders, mapByKey("name")) : [],
```

Reply:
sure, thanks @jonkoops